exercises/dnd-character/description.md

```diff
@@ -0,0 +1,34 @@
+For a game of [Dungeons & Dragons][DND], each player starts by generating a
+character they can play with. This character has, among other things, six
+abilities; strength, dexterity, constitution, intelligence, wisdom and
+charisma. These six abilities have scores that are determined randomly. You
+do this by rolling four 6-sided dice and record the sum of the highest three
+dice.  You do this six times, once for each ability.
+
+Your character's initial hitpoints are 10 + your character's constitution
+modifier. You find your character's constitution modifier by subtracting 10
+from your character's constitution, divide by 2 and round down.
+
+Write a random character generator that follows the rules above.
+
+Normally you would assign each dice roll to an ability freely, but for this
```

Comment:
"dice" is always plural. The singular is "die".
```suggestion
Normally you would assign each die roll to an ability freely, but for this
```

Reply:
Is it also called a die roll if there is more than one die?

Reply:
Yes, I'm pretty sure.